<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoViewer</class>
 <widget class="QDialog" name="InfoViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>348</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">
   <item>
    <layout class="QGridLayout" name="layoutInformation">
     <item row="1" column="0">
      <widget class="QLabel" name="titleLastName">
       <property name="text">
        <string>Last name:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="titleLastModification">
       <property name="text">
        <string>Last Modified:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="titleBirthday">
       <property name="text">
        <string>Birthday:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="titleFatherName">
       <property name="text">
        <string>Father's name:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="titleGender">
       <property name="text">
        <string>Gender:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="titleCreation">
       <property name="text">
        <string>Creation:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="titleBornProvince">
       <property name="text">
        <string>Born province:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="titleFirstName">
       <property name="text">
        <string>First name:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="titleNationality">
       <property name="text">
        <string>Nationality:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelFirstName">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labelLastName">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="labelFatherName">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="labelBirthday">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="labelBornProvince">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="labelGender">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="labelNationality">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="labelCreation">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="labelLastModification">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
